superset-frontend/src/filters/components/Select/buildQuery.test.ts

```diff
@@ -120,6 +120,8 @@ describe('Select buildQuery', () => {
     });
     expect(queryContext.queries.length).toEqual(1);
     const [query] = queryContext.queries;
-    expect(query.filters).toEqual([{ col: 'my_col', op: '>=', val: 123 }]);
+    expect(query.filters).toEqual([
+      { col: 'my_col', op: 'ILIKE', val: '%123%' },
```

Comment:
btw i think `LIKE` can be enough for since it matches only numbers, right?

Reply:
nvm. I don't find any performance benefit of using LIKE over ILIKE.

I'm good with it